commons/src/main/java/org/frankframework/util/ClassUtils.java

```diff
@@ -30,17 +30,16 @@
 import java.util.List;
 import java.util.Map;
 
-import org.apache.commons.lang3.StringUtils;
-import org.frankframework.core.INamedObject;
-
 import jakarta.annotation.Nonnull;
 import jakarta.annotation.Nullable;
```

Comment:
Dit is ook een system import, en mag een \n krijgen.

Reply:
`.editorconfig` en de intellij codestyle xml liepen uiteen.